static/src/js/components/GranuleResults/GranuleResultsHeader.js

```diff
@@ -142,6 +153,7 @@ class GranuleResultsHeader extends Component {
     const handoffLinks = generateHandoffs(metadata, collectionSearch, mapProjection)
 
     const initialLoading = ((pageNum === 1 && isLoading) || (!isLoaded && !isLoading))
+    // eslint-disable-next-line no-unused-vars
```

Comment:
unused variable? if so remove it

Reply:
Man. Youve got the eagle eye for these things.